data/thekrprince.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "Kumar Prince",
+  "type": "personal",
+  "bio": "Software Engineer at **Walmart**. Open Source enthusiast, Fitness freak and a Traveller",
+  "links": [
+    {
+      "name": "Twitter",
+      "url": "https://twitter.com/TheKrPrince",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/thekrprince/",
+      "icon": "FaLinkedIn"
```

You're probably be using an invalid icon here, It should be `FaLinkedin`.
```suggestion
      "icon": "FaLinkedin"
```